code/datums/abilities/wizard/teleport.dm

```diff
@@ -18,7 +18,7 @@
 		return 1
 
 // These two procs were so similar that I combined them (Convair880).
-/mob/proc/teleportscroll(var/effect = 0, var/perform_check = 0, var/obj/item_to_check = null, var/datum/targetable/spell/teleport/spell)
+/mob/proc/teleportscroll(var/effect = 0, var/perform_check = 0, var/obj/item_to_check = null, var/datum/targetable/spell/teleport/spell, var/abort_if_incapacitated = FALSE)
```

Comment:
I added this extra parameter to preserve the behavior for the brass gauntlet, teleport computer and the teleport spell.